html/pages/device/overview/processors.inc.php

```diff
@@ -16,44 +16,88 @@
     echo '</div>
         <table class="table table-hover table-condensed table-striped">';
 
+    $totalPercent=0;
+
     foreach ($processors as $proc) {
         $text_descr = rewrite_entity_descr($proc['processor_descr']);
 
         // disable short hrDeviceDescr. need to make this prettier.
         // $text_descr = short_hrDeviceDescr($proc['processor_descr']);
         $percent      = $proc['processor_usage'];
-        $background   = get_percentage_colours($percent);
-        $graph_colour = str_replace('#', '', $row_colour);
```

Comment:
Only the $background is used in both part, as actualy $graph_colour doesn't seem to be used.

But it make sense to compute the background colour on the final precentage and not on the last one.
I'll move this to the else block.

I need to clean up the variables that are not used.